ConsoleApp6/Program.cs

```diff
@@ -1,15 +1,41 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace ConsoleApp6
 {
-    class Program
+    public static class Program
     {
-        static void Main(string[] args)
+        public const int MAX_ATTEMPTS = 10;
+        public static void Main(string[] args)
         {
+            Console.WriteLine("\nOrdinary Garland: ");
+            OrdinaryGarland<Lamp> ordinaryGarland = new OrdinaryGarland<Lamp>();
+            Console.WriteLine("\nColored Garland: ");
+            ColoredGarland coloredGarland = new ColoredGarland();
+
+            ReturnStateOfOrdinaryOrColoredGarland(ordinaryGarland, coloredGarland);
+
+            Console.WriteLine("Work complete");
+            Console.ReadKey();
+        }
+
+        public static void ReturnStateOfOrdinaryOrColoredGarland(OrdinaryGarland<Lamp> ordinaryGarland, ColoredGarland coloredGarland)
```

Comment:
this method is incorrect. please, remove it and just call the ReturnCurrentStateOfLampsInGarland() method for each garland where it is required